File: ConnectUs/ConnectUs/www/scripts/objects/event.js
function Event(id, name, description, place, from, to) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.place = place;
    this.from = from;
    this.to = to;
    var now = new Date();
    if (this.from >= now && this.to <= now)
        this.state = 'Running';
    else if (this.from > now)
        this.state = 'Upcoming';
    else
        this.state = 'Finished';
}

File: ConnectUs/ConnectusMobileService/DataObjects/Event.cs
using Microsoft.WindowsAzure.Mobile.Service;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Data.Entity.Spatial;
using System.Linq;
using System.Web;

namespace ConnectusMobileService.DataObjects
{
    public class Event: EntityData
    {
        public Event()
        {
            this.Accounts = new HashSet<Account>();
        }

        [Required]
        [MaxLength(30)]
        public string Name { get; set; }
        public string Description { get; set; }
        [Required]
        [MaxLength(30)]
        public string Place { get; set; }
        public DbGeography Location { get; set; }

        [Required]
        public DateTimeOffset FromDate { get; set; }
        [Required]
        public DateTimeOffset ToDate { get; set; }
        public virtual ICollection<Account> Accounts { get; set; } //Participating users
    }
}